test-clients/src/itest/resources/network/config/bootstrap.properties

```diff
@@ -1 +1,4 @@
 bootstrap.throttleDefsJson.resource=throttles-dev.json
+hedera.recordStream.recordFileVersion=6
+hedera.recordStream.signatureFileVersion=6
+contracts.sidecars=CONTRACT_BYTECODE
```

Comment:
Q: will a separate PR see similar for state_changes and contract_actions?

Reply:
Correct, when we add the assertions (and tests) for actions and state changes, we will update the properties here so that the nodes in the CI generate them as well. 

We wanted to keep this PR as small as possible, consisting only of the infrastructure needed to start writing the tests/assertions and only adding the bytecode sidecar as proof that the whole plan is feasible. 

Will also make sure to update the PR description